modules/shortcodes/slideshow.php

```diff
@@ -279,14 +279,26 @@ public function enqueue_scripts() {
 			 * @since 4.7.0 Added the `speed` option to the array of options.
 			 *
 			 * @param array $args
-			 * - string - spinner - URL of the spinner image.
-			 * - string - speed   - Speed of the slideshow. Defaults to 4000.
+			 * - string - spinner        - URL of the spinner image.
+			 * - string - speed          - Speed of the slideshow. Defaults to 4000.
+			 * - string - class_prev     - CSS class for slideshow's previous button
+			 * - string - class_pause    - CSS class for slideshow's pause button
+			 * - string - class_next     - CSS class for slideshow's next button
+			 * - string - label_prev     - Aria label for slideshow's previous button
+			 * - string - label_pause    - Aria label for slideshow's pause button
+			 * - string - label_next     - Aria label for slideshow's next button
 			 */
 			apply_filters(
 				'jetpack_js_slideshow_settings',
 				array(
-					'spinner' => plugins_url( '/img/slideshow-loader.gif', __FILE__ ),
-					'speed'   => '4000',
+					'spinner'     => plugins_url( '/img/slideshow-loader.gif', __FILE__ ),
+					'speed'       => '4000',
+					'class_prev'  => 'classic-jetpack-slideshow_button-previous',
+					'class_pause' => 'classic-jetpack-slideshow_button-pause',
+					'class_next'  => 'classic-jetpack-slideshow_button-next',
+					'label_prev'  => 'Previous Slide',
```

Comment:
We still need these translated, a la:
```suggestion
					'label_prev'  => __( 'Previous Slide', 'jetpack' ),
```

Would need to be done for all three labels. What this does is allow the PHP-based process for l10n to work, which would then pass the translated strings to JS via `wp_localize_script`. Without the `__()` function, it would pass literally `Previous Slide` to JS without any modification.

Reply:
Ohh! Yes. I'll do that. Thanks, Kraft.

Reply:
This is done.